docs/pages/machine-id/guides/jenkins.mdx

```diff
@@ -0,0 +1,154 @@
+---
+title: Jenkins
+description: Using Teleport Machine ID with Jenkins
+---
+
+Jenkins is a open source automation server that is frequently used to build
+Continuous Integration and Continuous Delivery (CI/CD) pipelines.
+
+In this guide, we will demonstrate how to migrate existing Jenkins pipelines
+to utilize Machine ID with minimal changes.
+
+## Prerequisites
+
+You will need the following tools to use Teleport with Jenkins.
+
+- Teleport >= 9.0.0
+- `ssh` OpenSSH tool
+- Jenkins
+
+In addition, if you have not already done so, follow the
+[Machine ID Getting Started Guide](../../docs/pages/machine-id/getting-started.mdx)
+to create a bot user and start Machine ID.
+
+## Architecture
+
+Before we begin, it should be noted that Jenkins is a tool that is notoriously
+difficult to secure. Machine ID is one part of securing your infrastructure,
+but it alone is not sufficient. Below we will provide some basic guidance which
+can help improve the security posture of your Jenkins install.
+
+The simplest Jenkins deployments have the controller and agents run on the
```

Comment:
```suggestion
**Single-host deployments:** The simplest Jenkins deployments have the controller and agents run on the
```
Suggested adding bold catchphrases for each deployment method to help readers parse this part of the docs quickly